### PR TITLE
refactor and test voucher

### PR DESCRIPTION
**Pull request by fremartini**

### CODE REVIEW

File: test/features/voucher/data/datasources/voucher_remote_data_source_test.dart [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    executor = MockNetworkRequestExecutor();
+    apiV1 = MockCoffeecardApi();
+    remoteDataSource = VoucherRemoteDataSource(
+      apiV1: apiV1,
+      executor: executor,
+    );
+  });
+
+  group('redeemVoucher', () {
+    test('should call executor', () async {
```
Comment (marfavi) [inline test/features/voucher/data/datasources/voucher_remote_data_source_test.dart:28]: test name and assertion step differ ("Should call executor" would check if executor was called in the assertion step)

**Review by marfavi — CHANGES_REQUESTED** [review]

**Review by marfavi — DISMISSED** [review]

**Review by marfavi — DISMISSED** [review]
If you use the 'Update branch' button on GitHub, you won't invalidate any reviews :)

**Comment by fremartini** [comment]
@marfavi merged develop again. Can't Update Branch from here because merge conflicts

**Review by marfavi — APPROVED** [review]
All good

**Pull request merged**